test/cli/attest_cmd_test.go

```diff
@@ -0,0 +1,50 @@
+package cli
+
+import (
+	"strings"
+	"testing"
+)
+
+func TestAttestCmdFlags(t *testing.T) {
```

Comment:
a great addition to this test would be to call out to `cosign` and verify the output with the [flow you added in the readme](https://github.com/anchore/syft#adding-an-sbom-to-an-image-as-an-attestation)